src/app/components/blogs/blogs.component.scss

```diff
@@ -2,6 +2,6 @@
 @import '../../styles/placeholders/placeholders.scss';
 @import '../../styles/variables/variables.scss';
 
-.blogs-container {
-    @extend %columnSpaceBetween;
-}
\ No newline at end of file
+// .blogs-container {
```

Comment:
remove commented code

Reply:
remove commented code and unnecessary imports